ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_trigonometric_functions.py

```diff
@@ -332,3 +332,51 @@ def test_numpy_deg2rad(
         dtype=dtype,
         subok=True,
     )
+
+
+# degrees
+@handle_frontend_test(
+    fn_tree="numpy.degrees",
+    dtypes_values_casting=np_frontend_helpers.dtypes_values_casting_dtype(
+        arr_func=[
+            lambda: helpers.dtype_and_values(
+                available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
for example, in this case, I think it by default should only accept float arguments, so this should be `float` and not `numeric`